plugin/src/main/java/com/denizenscript/denizen/Denizen.java

```diff
@@ -603,9 +605,10 @@ private boolean suggestClosestModifier(CommandSender sender, String command, Str
 
     @Override
     public ChunkGenerator getDefaultWorldGenerator(String worldName, String id) {
-        if (CoreUtilities.toLowerCase(id).equals("void")) {
-            return new VoidGenerator1_17();
-        }
-        return null;
+        return switch (CoreUtilities.toLowerCase(id)) {
+            case "void" -> new VoidGenerator(true);
+            case "void_biomes" -> new VoidGenerator(false);

```

Comment:
Should probably have a `void:<BiomeTag>` rather than a boolean for void biome or nonsense biome